targets/TARGET_NXP/TARGET_LPC176X/TARGET_MBED_LPC1768/PinNames.h

```diff
@@ -106,11 +106,17 @@ typedef enum {
     A4 = P1_30,
     A5 = P1_31,
 
-    I2C_SCL = D15,
-    I2C_SDA = D14,
-
     // Not connected
-    NC = (int)0xFFFFFFFF
+    NC = (int)0xFFFFFFFF,
+
+    I2C_SCL0 = NC,
+    I2C_SDA0 = NC,
+    I2C_SCL1 = p10,
+    I2C_SDA1 = p9,
+    I2C_SCL2 = p27, // pin used by application board
+    I2C_SDA2 = p28, // pin used by application board
+    I2C_SCL = I2C_SCL2,
+    I2C_SDA = I2C_SDA2,
```

Comment:
I'm pretty sure this is going to be a breaking change. `I2C_SCL` used to follow this map `I2C_SCL->D15->P0_28`. After these changes, `I2C_SCL` will follow this map: `I2C_SCL->I2C_SCL2->p27->P0_11`. `I2C_SDA` is also following a different mapping. This will break any current application that use the `I2C_SCL` and `I2C_SDA` pin names.